CSV_File_Reading/src/de/telran/MethodModule.java

```diff
@@ -0,0 +1,38 @@
+package de.telran;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class MethodModule {
+
+    public static List<Person> readingACSVFileToAList(File csvFile) {
+        List<Person> persons = new ArrayList<>();
+        String line;
+
+        try (BufferedReader br = new BufferedReader(new FileReader(csvFile))) {
+            while ((line = br.readLine()) != null) {
+                String[] person = line.split(",");
```

Comment:
Making a Person out of a String could be encapsulated into a separate method, which is then referred by reference in the map() method.